models/README.md

```diff
@@ -7,3 +7,27 @@ servers. These diffs are generated against the standalone-full-ha.xml profile, w
 
 This data allows the Wildscribe pages to be generated offline, without a running server instance. It also means that at some
 stage it should be possible to generate diff's of the models.
+
+Add the XTS subsystem
+=====================
+Add the following to standalone-full-ha.xml:
+
+  <extension module="org.jboss.as.xts"/>
+
+  <subsystem xmlns="urn:jboss:domain:xts:2.0">
+      <host name="default-host"/>
+      <xts-environment url="http://${jboss.bind.address:127.0.0.1}:8080/ws-c11/ActivationService"/>
+      <default-context-propagation enabled="true"/>
+  </subsystem>
```

Comment:
Sorry, probably a dumb comment ^^^, as we just need the subsystem description, and the specific config should be irrelevant to that.

I'm curious why the subsystem config is needed at all; i.e. simply the extension element is enough to register the subsystem API.

Do we need rts (see standalone-rts.xml) and picketlink? IIRC the config in standalone-picketlink.xml won't actually boot though. It's more of a form of documentation.